vllm/distributed/parallel_state.py

```diff
@@ -342,6 +346,62 @@ def broadcast_object_list(self,
                                                 group=self.device_group)
         return obj_list
 
+    def send_object(self,
+                    obj: Any,
+                    dst: int,
+                    group: Optional[ProcessGroup] = None) -> None:
+        """Send the input object list to the destination rank."""
+        assert dst < self.world_size, f"Invalid dst rank ({dst})"
+
+        assert dst != self.rank, (
+            "Invalid destination rank. Destination rank is the same "
+            "as the current rank.")
+
+        current_device = _get_pg_default_device(group)
+        # Serialize object to tensor.
+        object_tensor, size_tensor = _object_to_tensor(obj, current_device,
```

Comment:
safe to just use `pickle.dumps` and `torch.frombuffer` stuff.

Reply:
Done these simplifications